components/output/Container.vue

```diff
@@ -12,8 +14,17 @@ const { data, status, error } = useAsyncData(
       context = await bundler.init()
       bundler.initted = true
     }
+
     const configObject = new Function(config.value)()
+    const startTime = performance.now()
     const result = await bundler.build.call(context, code.value, configObject)
+    const endTime = performance.now()
+
+    if (!isInitialExecution.value) {
+      lastBuildTime.value = Math.round(endTime - startTime)
```

Comment:
```suggestion
      lastBuildTime.value = Math.round(performance.now() - startTime)
```
`endTime` can be inline.